.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Could you kindly elaborate what is the purpose of this file?

Reply:
It was automatically updated with the text editor, vscode which was unnecessary. Unnecessary files are now removed.